sdk/storage/storage-blob/package.json

```diff
@@ -82,7 +83,7 @@
     "prebuild": "npm run clean",
     "pretest": "npm run build:test",
     "test:browser": "karma start --single-run",
-    "test:node": "cross-env TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\": \\\"commonjs\\\"}\" nyc mocha --compilers ts-node/register --require source-map-support/register --reporter mocha-multi-reporters --reporter-options configFile=mocha.reporter.config.json --full-trace --no-timeouts test/*.test.ts test/node/*.test.ts",
+    "test:node": "nyc mocha --reporter mocha-multi-reporters --reporter-options configFile=mocha.reporter.config.json -t 120000 dist-test/index.node.js",
```

Comment:
When we test against a single bundle file, failed case reported stack trace is not friendly (not pointing to original TS files), makes it difficulty to locate to specific line for debugging. Especially to debug failed cases reported in CIs. Any good suggestions?

Reply:
That's a good question. But currently, we don't have a way.
(Even for other packages, reported stack trace doesn't point to the original TS files).
I can start a new issue to track this.


[On a side note, a single bundle file would help us enable VS Code debugging effortlessly.]

Reply:
Aha, how did you debug in VS code with a single bundle file? will break point work properly? I previously use node-ts, and it works great especial for break points debugging.

Reply:
It will point to the source files(typescript) using the source mappings.
We can set breakpoints in the typescript source files.
Created a draft PR for VS Code Debugging - 
https://github.com/Azure/azure-sdk-for-js/pull/2770/files#diff-8b42a0e706358bbff6b0a06ac092e5ac. 
[Contains changes from this PR, can only be merged after this PR for rollup]

Reply:
thanks, that's very useful!

Reply:
Source maps with vscode are pretty sweet. But there is also this library we could investigate as a dev dependency: https://github.com/evanw/node-source-map-support. It maps all the file/offset data in exception payloads using sourcemaps.

Reply:
Thanks Brian, creating a new issue to track this - https://github.com/Azure/azure-sdk-for-js/issues/2792